clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: machine.openshift.io/v1beta1
```

Comment:
I don't understand this approach.

What we want is to scale up/down the us-east-1b-m5d machineset to see if the machines become nodes or not.

Reply:
This is our production. Since we could have problem with `m5d` autoscaling.
We need to recover the autoscaling on `m5` instance in case that the cluster needs more nodes.